309thEDDGE/opal-helm
# Add prometheus

Validation setup:
- Install the chart at https://github.com/309thEDDGE/kube-prometheus to the `monitoring` namespace with deployment name `prometheus`
- Port-forward the prometheus-kube-prometheus-prometheus service to port 9090. 

There are a few indications that prometheus has successfully connected to minio. 
1. With the prometheus port forward active, check http://localhost:9090/targets for an entry called `kubernetes-pods`. Click `show more`, and you should see four entries pointing to <ip address>:9000/minio/v2/metrics/cluster
2. Log in to the minio dashboard, and go to monitoring -> metrics. You should be able to select each tab without minio throwing an error that prometheus cannot be reached
